### PR TITLE
dotnet -Update proposals (adds 7)

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Addresses - https://github.com/devcontainers/templates/issues/38
Depends upon -
- https://github.com/devcontainers/images/pull/201

### CODE REVIEW

**Comment by Chuxel** [comment]
We can also add 6.0-jammy given the tweak I just did for https://github.com/devcontainers/images/pull/201

File: src/dotnet/devcontainer-template.json [hunk 1]
```diff
@@ -11,14 +11,18 @@
...
+                "7.0-bullseye",
                 "6.0-bullseye",
                 "3.1-bullseye",
+                "7.0-jammy",
+                "6.0-jammy",
                 "6.0-focal",
                 "3.1-focal"
             ],
-            "default": "6.0-bullseye"
+            "default": "7.0-bullseye"
```
Comment (jkeech) [inline src/dotnet/devcontainer-template.json:25]: Can we simplify this to just `7.0`? I feel like the default value should be very simple with the more specific distro choice available as an option. Most users probably care more about the .NET SDK/runtime version and not as much about the underlying distro version they are developing on (and if they do care, they can select a more specific option).

File: src/dotnet/devcontainer-template.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "id": "dotnet",
-    "version": "1.0.2",
+    "version": "1.0.3",
```
Comment (jkeech) [inline src/dotnet/devcontainer-template.json:3]: Should we bump this to 1.1.0 to follow more sem-ver conventions? Adding the new .NET 7 version as the default seems like a new feature worth a minor version bump.

**Review by jkeech — DISMISSED** [review]
Approved with a couple suggestions.

**Review by joshspicer — APPROVED** [review]

File: src/dotnet/devcontainer-template.json (hunk 1 above)
Comment (samruddhikhandale) [reply src/dotnet/devcontainer-template.json]: Makes sense. Updated

**Pull request merged**